tests/bench/coremark/coremark_main.c

```diff
@@ -303,7 +323,9 @@ for (i = 0; i < MULTITHREAD; i++)
         results[0].iterations *= 1 + 10 / divisor;
     }
     /* perform actual benchmark */
+#ifndef CONFIG_COREMARK_SIMPLE_REPORT
     start_time();
+#endif
```

Comment:
It reduces code readability in a function by adding too many #ifdef .
The reasonable implementation is:
#ifdef CONFIG_CORE_MARK_SIMPLE_REPORT
void start_time(void)
{
...
}
#else 
#define start_time() do { } while (0)
#endif 

So that stubs are maintain at function/macro boundary and the main function flow can be kept neat.